packages/canary-web-components/src/components/ic-date-picker/ic-date-picker.tsx

```diff
@@ -0,0 +1,1511 @@
+import {
+  Component,
+  Element,
+  Host,
+  h,
+  Prop,
+  Listen,
+  Watch,
+  State,
+  Event,
+  EventEmitter,
+} from "@stencil/core";
+import {
+  createDate,
+  clampDate,
+  dateMatches,
+  dateInRange,
+  getMonthStart,
+  getMonthEnd,
+  getWeekEnd,
+  getWeekStart,
+  yearInRange,
+} from "../../utils/date-helpers";
+import {
+  stringEnumToArray,
+  onComponentRequiredPropUndefined,
+} from "../../utils/helpers";
+import {
+  IcWeekDays,
+  IcShortDayNames,
+  IcDateInputMonths,
+  IcDateFormat,
+  IcSizes,
+  IcInformationStatusOrEmpty,
+} from "../../utils/types";
+import chevron from "../../assets/chevron-icon.svg";
+import { DayButton } from "./ic-day-button";
+import { MonthPicker } from "./ic-month-picker";
+import { YearPicker } from "./ic-year-picker";
+
+const DEFAULT_DATE_FORMAT = "DD/MM/YYYY";
+const DEFAULT_DISABLE_DATES_FROM_NOW_MSG =
+  "Dates in the future are not allowed. Please select a date in the past.";
+const DEFAULT_DISABLE_DATES_UNTIL_NOW_MSG =
+  "Dates in the past are not allowed. Please select a date in the future.";
+const DEFAULT_DISABLE_DAYS_MSG =
+  "The date you have selected is on a day of the week that is not allowed. Please select another date.";
```

Comment:
More of a question as i'm not sure if it's possible, but can these be used as the default values for the props as well e.g. `@Prop() disableDaysMessage?: string = DEFAULT_DISABLE_DAYS_MSG`

Reply:
afraid not, it causes storybook to show the variable name in the table